Ginger/Ginger/SolutionWindows/TreeViewItems/ApplicationModelsTreeItems/AppApiModelsFolderTreeItem.cs

```diff
@@ -171,5 +172,38 @@ public void AddAPIModelFromDocument(object sender, RoutedEventArgs e)
         {            
             WizardWindow.ShowWizard(new AddAPIModelWizard(mAPIModelFolder), 1000);
         }
+
+        public override bool PasteCopiedTreeItem(object nodeItemToCopy, TreeViewItemGenericBase targetFolderNode, bool toRefreshFolder = true)
+        {
+            if (nodeItemToCopy is RepositoryItemBase)
+            {
```

Comment:
instead repeating the same code twice, create STATIC HandleAPIItemDuplication(APIModel source) method which returns the duplicated API model on the TreeItem class and then call it also from FolderTree Item for Paste- so then we will need to maintain only one code